Library/Homebrew/cmd/install.rb

```diff
@@ -143,7 +145,12 @@ def check_cellar
   def perform_preinstall_checks
     check_ppc
     check_writable_install_location
-    check_xcode
+    if MacOS::Xcode.installed?
+      check_xcode
+    else
+      opoo "You have not installed Xcode."
+      puts "Bottles may install correctly, but builds will fail!"
```

Comment:
Yeah, that makes sense. I'll remove it as I implement the dependency bottle checking system.